<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <property name="spacing">
    <number>11</number>
   </property>
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>90</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#groupBox{
border:none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="PixmapLabel" name="icon_photo"/>
           </item>
           <item>
            <widget class="BodyLabel" name="BodyLabel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Agency FB</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string> Create your account</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="LineEdit" name="LineEdit_4">
          <property name="styleSheet">
           <string notr="true">LineEdit, TextEdit, PlainTextEdit {
    color: gray;
    background-color: rgba(255, 255, 255, 0.7);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 100);
    border-radius: 5px;
    /* font: 14px &quot;Segoe UI&quot;, &quot;Microsoft YaHei&quot;; */
    padding: 0px 10px;
    selection-background-color: #00a7b3;
}

TextEdit,
PlainTextEdit {
    padding: 2px 3px 2px 8px;
}

LineEdit:hover, TextEdit:hover, PlainTextEdit:hover {
    background-color: rgba(249, 249, 249, 0.5);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 100);
}

LineEdit:focus {
    border-bottom: 1px solid rgba(0, 0, 0, 13);
    background-color: white;
}

TextEdit:focus,
PlainTextEdit:focus {
    border-bottom: 1px solid #009faa;
    background-color: white;
}

LineEdit:disabled, TextEdit:disabled,
PlainTextEdit:disabled {
    color: rgba(0, 0, 0, 150);
    background-color: rgba(249, 249, 249, 0.3);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 13);
}

#lineEditButton {
    background-color: transparent;
    border-radius: 4px;
    margin: 0;
}

#lineEditButton:hover {
    background-color: rgba(0, 0, 0, 9);
}

#lineEditButton:pressed {
    background-color: rgba(0, 0, 0, 6);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEdit" name="LineEdit_2">
          <property name="styleSheet">
           <string notr="true">LineEdit, TextEdit, PlainTextEdit {
    color: gray;
    background-color: rgba(255, 255, 255, 0.7);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 100);
    border-radius: 5px;
    /* font: 14px &quot;Segoe UI&quot;, &quot;Microsoft YaHei&quot;; */
    padding: 0px 10px;
    selection-background-color: #00a7b3;
}

TextEdit,
PlainTextEdit {
    padding: 2px 3px 2px 8px;
}

LineEdit:hover, TextEdit:hover, PlainTextEdit:hover {
    background-color: rgba(249, 249, 249, 0.5);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 100);
}

LineEdit:focus {
    border-bottom: 1px solid rgba(0, 0, 0, 13);
    background-color: white;
}

TextEdit:focus,
PlainTextEdit:focus {
    border-bottom: 1px solid #009faa;
    background-color: white;
}

LineEdit:disabled, TextEdit:disabled,
PlainTextEdit:disabled {
    color: rgba(0, 0, 0, 150);
    background-color: rgba(249, 249, 249, 0.3);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 13);
}

#lineEditButton {
    background-color: transparent;
    border-radius: 4px;
    margin: 0;
}

#lineEditButton:hover {
    background-color: rgba(0, 0, 0, 9);
}

#lineEditButton:pressed {
    background-color: rgba(0, 0, 0, 6);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>邮箱</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEdit" name="LineEdit">
          <property name="styleSheet">
           <string notr="true">LineEdit, TextEdit, PlainTextEdit {
    color: gray;
    background-color: rgba(255, 255, 255, 0.7);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 100);
    border-radius: 5px;
    /* font: 14px &quot;Segoe UI&quot;, &quot;Microsoft YaHei&quot;; */
    padding: 0px 10px;
    selection-background-color: #00a7b3;
}

TextEdit,
PlainTextEdit {
    padding: 2px 3px 2px 8px;
}

LineEdit:hover, TextEdit:hover, PlainTextEdit:hover {
    background-color: rgba(249, 249, 249, 0.5);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 100);
}

LineEdit:focus {
    border-bottom: 1px solid rgba(0, 0, 0, 13);
    background-color: white;
}

TextEdit:focus,
PlainTextEdit:focus {
    border-bottom: 1px solid #009faa;
    background-color: white;
}

LineEdit:disabled, TextEdit:disabled,
PlainTextEdit:disabled {
    color: rgba(0, 0, 0, 150);
    background-color: rgba(249, 249, 249, 0.3);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 13);
}

#lineEditButton {
    background-color: transparent;
    border-radius: 4px;
    margin: 0;
}

#lineEditButton:hover {
    background-color: rgba(0, 0, 0, 9);
}

#lineEditButton:pressed {
    background-color: rgba(0, 0, 0, 6);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PasswordLineEdit" name="PasswordLineEdit">
          <property name="placeholderText">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>4</width>
            <height>6</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#groupBox_4{
border:none;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="LineEdit" name="LineEdit_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>6</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>10000000</width>
               <height>33</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">LineEdit, TextEdit, PlainTextEdit {
    color: gray;
    background-color: rgba(255, 255, 255, 0.7);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 100);
    border-radius: 5px;
    /* font: 14px &quot;Segoe UI&quot;, &quot;Microsoft YaHei&quot;; */
    padding: 0px 10px;
    selection-background-color: #00a7b3;
}

TextEdit,
PlainTextEdit {
    padding: 2px 3px 2px 8px;
}

LineEdit:hover, TextEdit:hover, PlainTextEdit:hover {
    background-color: rgba(249, 249, 249, 0.5);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 100);
}

LineEdit:focus {
    border-bottom: 1px solid rgba(0, 0, 0, 13);
    background-color: white;
}

TextEdit:focus,
PlainTextEdit:focus {
    border-bottom: 1px solid #009faa;
    background-color: white;
}

LineEdit:disabled, TextEdit:disabled,
PlainTextEdit:disabled {
    color: rgba(0, 0, 0, 150);
    background-color: rgba(249, 249, 249, 0.3);
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 13);
}

#lineEditButton {
    background-color: transparent;
    border-radius: 4px;
    margin: 0;
}

#lineEditButton:hover {
    background-color: rgba(0, 0, 0, 9);
}

#lineEditButton:pressed {
    background-color: rgba(0, 0, 0, 6);
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>验证码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ImageLabel" name="veri_code">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>72</width>
               <height>6</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>33</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">#groupBox_2{
	border:none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="PushButton" name="confirm_button">
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PushButton" name="cancel_button">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PixmapLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PasswordLineEdit</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
